packages/web/src/scss/components/Modal/_ModalDialog.scss

```diff
@@ -1,189 +1,88 @@
 //    the tablet breakpoint. However, `top` and `bottom` are not transitioned when switching from and to `auto`.
-// 2. Use ScrollView's backdoor to fix its height inside ModalDialog in Safari.
-// 3. We need to set the box-sizing again because the parent element unsets styles using `all: unset`.
-// 4. Suppress bottom alignment of the docked variant and use the alignment variants of `.Modal` instead.
-// 5. Unfortunately, the open state cannot be part of the parent `-docked-modal-dialog()` mixin because it's not
-//    possible to generate selector for the feature class scenario.
-// 6. Override bottom padding of parent `.Modal` in the docked variant.
-// 7. Override the min-height of the expanded docked variant.
+// 4. Override bottom padding of parent `.Modal` in the docked variant.
 
 @use 'sass:map';
 @use '../../settings/feature-flags';
 @use '../../tools/breakpoint';
 @use '../../tools/typography';
 @use 'theme';
 
-@mixin -modal-dialog() {
+.ModalDialog {
     @include typography.generate(theme.$typography);
 
-    --scroll-view-vertical-height: auto; // 2.
+    --scroll-view-vertical-height: auto; // 1.
 
     display: flex;
     flex-direction: column;
-    box-sizing: border-box; // 3.
+    box-sizing: border-box; // 2.
     width: theme.$dialog-width;
+    max-width: calc(100% - #{theme.$padding-x});
+    height: theme.$dialog-height;
+    max-height: theme.$dialog-max-height;
     margin-inline: auto;
     margin-top: var(--modal-top);
     margin-bottom: var(--modal-bottom);
     overflow-x: var(--modal-body-overflow-x, hidden);
-    overflow-y: var(--modal-body-overflow-x, auto);
+    overflow-y: var(--modal-body-overflow-y, auto);
     color: theme.$dialog-text-color;
+    border-radius: theme.$dialog-border-radius;
     background-color: theme.$dialog-background-color;
     box-shadow: theme.$dialog-shadow;
     transform: translateY(var(--modal-translate-y)) scale(var(--modal-scale));
-    overscroll-behavior: contain;
     transform-origin: var(--modal-transform-origin);
-
-    @media (prefers-reduced-motion: no-preference) {
-        transition-property: width, height, max-height, border-radius, transform; // 1.
-        transition-duration: inherit;
-    }
-}
-
-@mixin -uniform-modal-dialog($from-breakpoint) {
-    @include breakpoint.up($from-breakpoint) {
-        // 4.
-        @if $from-breakpoint == 0 {
-            --modal-top: unset;
-            --modal-bottom: unset;
-            --modal-translate-y: unset;
-            --modal-transform-origin: unset;
-        }
-
-        width: theme.$dialog-uniform-width;
-        max-width: calc(100% - #{theme.$padding-x});
-        height: theme.$dialog-uniform-height;
-        min-height: unset; // 7. @deprecated Can be removed once the uniform modal dialog has been made default.
-        border-radius: theme.$dialog-border-radius;
-    }
+    overscroll-behavior: contain;
```

Comment:
apparently, we don't have this property in stylelint-order, because it was before overscroll before :)